test/test_class_trel.js

```
@@ -0,0 +1,382 @@
+      utils: {
+        log: jest.fn(),
+      },
+      trelloApiKey: 'test-api-key',
+    };
+
+    // Load and evaluate Trel class
+    const trelCode = require('fs').readFileSync(
+      'chrome_manifest_v3/class_trel.js',
+      'utf8',
+    );
+    eval(trelCode);
```

Biome (2.1.2)

[error] eval() exposes to security risks and performance issues.

See the MDN web docs for more details.
Refactor the code so that it doesn't need to call eval().

(lint/security/noGlobalEval)